src/db/util/raw-model.ts

```diff
@@ -45,6 +46,14 @@ export type FindFn<F extends FieldDefinition, AdditionalArgs = {}> = (
      */
     skipValidation?: boolean;
     trx?: Knex.Transaction<any, any>;
+    /**
+     * Uses `DISTINCT ON`, which is a feature unique to PostgreSQL.
+     *
+     * Keeps only the first row of each set of rows where the given (set of) column(s) is unique.
+     * Note that the “first row” of each set is unpredictable unless `ORDER BY` is used to
+     * ensure that the desired row appears first.
+     */
```

Comment:
I know it was my suggestion to add this docstring, but now I realize there is an important constraint related to ordering that we should document
```suggestion
    /**
     * Uses `DISTINCT ON`, which is a feature unique to PostgreSQL.
     *
     * Keeps only the first row of each set of rows where the given (set of) column(s) is unique.
     * Note that the “first row” of each set is unpredictable unless `ORDER BY` is used to
     * ensure that the desired row appears first. If ordering is used, there is an important
     * constraint, because the first expression of `ORDER BY` must be present **somewhere** in
     * `DISTINCT ON` expressions. But, not all expressions of `ORDER BY` need to be in `DISTINCT ON`.
     */
```

Reply:
Done!